recipe/meta.yaml

```diff
@@ -1,32 +1,174 @@
 {% set version = "3.0.0" %}
+{% set jpeg_so_version = "8.3.2" %}
+{% set jpeg_so_major_version = jpeg_so_version.split('.')[0] %}
+{% set turbo_so_version = "0.3.0" %}
+{% set turbo_so_major_version = turbo_so_version.split('.')[0] %}
 
 package:
-  name: libjpeg-turbo
+  name: libjpeg-turbo-split
   version: {{ version }}
 
 source:
   url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/{{ version }}.tar.gz
   sha256: 171dae5d73560bc94006a7c0c3281bd9bfde6a34f7e41e66f930a1a9162bd7df
 
 build:
-  number: 1
-  run_exports:
-    - {{ pin_subpackage('libjpeg-turbo', max_pin='x') }}
+  number: 2
 
 requirements:
   build:
     - {{ compiler('c') }}
     - cmake
     - make
     - yasm
-  run_constrained:
-    - jpeg <0.0.0a
-
-test:
-  files:
-    - testorig.jpg
-  commands:
-    - djpeg -dct int -ppm -outfile testout.ppm testorig.jpg
+
+outputs:
+
+  # Binary package; this is the exported name from the pre-multi-output recipe
+  - name: libjpeg-turbo
```

Comment:
Because this output contains all binaries, named-libraries, manpages, and examples no downstream packages should break. However, downstream recipes will need to migrate themselves to `libjpeg-turbo-dev` the next time they are built.